MAVProxy/modules/mavproxy_chat/chat_window.py

```diff
@@ -17,6 +17,9 @@ def __init__(self, mpstate, wait_for_command_ack_fn):
         # keep reference to mpstate
         self.mpstate = mpstate
 
+        # keep the recording state
+        self.isRecording = False
```

Comment:
let's change this to is_recording.  We normally use lower case and underscores

Reply:
Sure! In-fact I just took a look, I think its no longer needed, will fix it.